src/components/body/content/categories/Categories.tsx

```diff
@@ -1,131 +1,54 @@
 import "./index.scss";
-import { FormattedMessage, useIntl } from "react-intl";
-import CategoryBusiness from "../../../../assets/category_business.png";
-import CategoryDesign  from "../../../../assets/category_design.png";
-import CategoryIT  from "../../../../assets/category_it.png";
-import CategoryMusic  from "../../../../assets/category_music.png";
+import { useEffect, useState } from "react";
+import { BACKEND_URL } from "../../../../constants";
+import { FormattedMessage } from "react-intl";
+import SkeketonCategory from "../../../SkeletonCategory";
 import UserCategory from "../../../UserCategory";
+import axios from "axios";
 
-// eslint-disable-next-line max-len
-const TextIT = "The newest and most useful video tutorials on programming and high technologies will help you understand all the variety of information. Video lessons on specific technologies contribute to the development of the necessary skills. Explore the rabbit hole with experts in web development, programming and robotics";
-// eslint-disable-next-line max-len
-const TextDesign = "Design education: 30+ professions and 1000+ courses in graphic, web design, UX and UI, product and industrial design. Current approaches to learning. Opportunity to study disciplines from scratch and improve your skills";
-// eslint-disable-next-line max-len
-const TextMusic = "Videos and trainings on all aspects of music: harmony, theory, improvisation, arrangement, composition, melody creation, song creation, guitar. The opportunity to try yourself in different genres and directions. Learn from scratch and improve your skills with the best professionals in the music industry";
-// eslint-disable-next-line max-len
-const TextBusiness = "Discover a large collection of videos on the topic of business and entrepreneurship, that help you to learn the basics of financial literacy and help you organize your business processes. Thousands of experts share their knowledge of economics, management, business analysis, and how to organize your own business";
+const categoriesUrl = `${BACKEND_URL}/categories`;
+export interface Category {
+  id: number;
+  image_url: string;
+  name: string;
+  description: string;
+}
 
 const Categories = () => {
-  const intl = useIntl();
+  const [categories, setCategories] = useState<Category[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  async function fetchCategory() {
+    try {
+      const {data} = await axios.get(categoriesUrl,);
+      setCategories(data);
+      setIsLoading(false);
+    } catch (error) {
+      alert("Error getting categories!!!");
+    }
+  }
+ 
+  useEffect(() => {
+    fetchCategory();
+  }, []);
+  
+  const categorySet = categories.map((obj) => (
+    <UserCategory
+      key={obj.id}
+      id={obj.id}
+      imagePreview={obj.image_url}
+      name={obj.name} 
+      description={obj.description} 
+    />
+  ));
+
+  const skeleton = [...new Array(19)].map((_, index) => <SkeketonCategory key={index} />);
```

Comment:
move 19 - to constant to the top of the file - it will be more easy to find and change it for others

Reply:
ok